v1.0/functions-and-operators.md

```diff
@@ -33,8 +33,8 @@ The following table lists all CockroachDB operators from highest to lowest prece
 |  | `>>` | Bitwise right-shift | binary |
 | 8 | `&` | Bitwise and | binary |
 | 9 | `#` | Bitwise xor | binary |
-| 10 | `|` | Bitwise or | binary |
-| 11 | `||` | Concatenation | binary |
+| 10 | <code>&#124;</code> | Bitwise or | binary |
```

Comment:
Does `<code>|</code>` work as well? If so I'd prefer that.

Reply:
Nope. 

<img width="826" alt="screen shot 2017-09-14 at 12 26 13 am" src="https://user-images.githubusercontent.com/15893490/30412116-58e7a870-98e3-11e7-9192-2d16258e343d.png">

Reply:
This is a bug in redcarpet: https://github.com/vmg/redcarpet/issues/477

Reply:
Sad.

Reply:
Agreed. Hoping we can explore reStructuredText soonish.